### PR TITLE
Add stub of ops page

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Review by lachlanjc — APPROVED** [review]
THANK YOU for doing this 💖

File: src/pages/admin/clubs/edit.js [hunk 1]
```diff
@@ -1,12 +1,130 @@
...
+            this.setState({ isLoading: false })
+            alert('User not found')
+          })
+      } else {
+        this.setState({ isLoading: false })
+        alert('Unable to find user with that email')
+      }
+    })
+  }
+  handleSubmit = this.handleSubmit.bind(this)
```
Comment (lachlanjc) [inline src/pages/admin/clubs/edit.js]: Use handleSubmit = e => when defining the method and you can remove this line ✨

File: src/pages/admin/clubs/edit.js [hunk 2]
```diff
@@ -1,12 +1,130 @@
...
+      return (
+        <Fragment>
+          <Text>Club owner unset.</Text>
+          <form onSubmit={this.handleSubmit}>
+            <Autocomplete
+              getItemValue={item => item.label}
+              items={cache[value] || []}
+              renderInput={props => <input {...props} width="32px" />}
+              renderItem={(item, isHighlighted) => (
+                <div
```
Comment (lachlanjc) [inline src/pages/admin/clubs/edit.js]: <Box bg= 🙂

File: src/pages/admin/clubs/edit.js (hunk 1 above)
Comment (maxwofford) [reply src/pages/admin/clubs/edit.js]: Good idea, thanks!

File: src/pages/admin/clubs/edit.js (hunk 2 above)
Comment (maxwofford) [reply src/pages/admin/clubs/edit.js]: 🌻

**Pull request merged**